src/modules/fancyzones/lib/JsonHelpers.cpp

```diff
@@ -938,15 +270,15 @@ namespace JSONHelpers
         return result;
     }
 
-    std::optional<ZoneSetData> ZoneSetData::FromJson(const json::JsonObject& zoneSet)
+    std::optional<FancyZonesDataTypes::ZoneSetData> ZoneSetDataJSON::FromJson(const json::JsonObject& zoneSet)
     {
         try
         {
-            ZoneSetData zoneSetData;
+            FancyZonesDataTypes::ZoneSetData zoneSetData;
             zoneSetData.uuid = zoneSet.GetNamedString(L"uuid");
-            zoneSetData.type = TypeFromString(std::wstring{ zoneSet.GetNamedString(L"type") });
+            zoneSetData.type = FancyZonesDataTypes::TypeFromString(std::wstring{ zoneSet.GetNamedString(L"type") });
```

Comment:
Replace all strings literals with const strings and explicitly mark them as non-localizable.